server/mongo.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
```

Comment:
please left **TODO: refactor** here.
we will deal with this later in separate PR (Pull Request)

Reply:
Done